helm-chart/images/image-cleaner/requirements.txt

```
@@ -1,2 +1,2 @@
-docker==3.2.1
-kubernetes==3.0.0
+docker==4.1.0
+kubernetes==10.0.0
```

10.0.1 contains a security fix

```suggestion
kubernetes==10.0.1
```